examples/cql-examples.json

```diff
@@ -65,6 +65,10 @@
     {
       "name": "bin_maxbins",
       "title": "Histogram - maxbins?"
+    },
+    {
+      "name": "scaleType",
```

Comment:
Now I wonder if we should rename the example files:

`bin_maxbins` => `bin-maxbins`
`scaleType` => `scaleType`

for consistency with the rest of examples.  Any thoughts?